packages/explorer/src/app/(explorer)/[chainName]/worlds/WorldsForm.tsx

```diff
@@ -25,6 +27,9 @@ const formSchema = z.object({
 });
 
 export function WorldsForm({ worlds }: { worlds: Address[] }) {
+  // Initialize the observer store to start fetching transactions
+  useStore(store);
```

Comment:
just had a thought: is next.js just looking for a reference to the import in order to consider it used/in scope of the bundle? if so, I wonder if this would solve our problems

```suggestion
  store;
```

Reply:
haha didn't think about that, def on the weirder side of code that I've seen but yeah, it does work. So perhaps let's stick with that